package.json

```diff
@@ -1,16 +1,21 @@
 {
-  "name": "gci-leaders",
-  "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
-  "scripts": {
-    "build": "node scrap.js && node generate.js"
-  },
-  "keywords": [],
-  "author": "blazeu <neversleepman@tfwno.gf>",
-  "license": "MIT",
-  "dependencies": {
-    "mustache": "^2.3.0",
-    "node-fetch": "^1.7.3"
-  }
```

Comment:
Keep the indentation from npm (2 spaces).

I know npm@5 does read the indentation and use that,
but if we keep the previous style, we don't need to modify this file :)